github-wide.css

```diff
@@ -120,3 +120,8 @@ button.discussion-sidebar-toggle {
 .vcard-avatar {
   width: 230px !important;
 }
+
+/* Commits: extended message under "..." */
+.commit-desc pre {
+  max-width: inherit;
```

Comment:
What exactly does this compute to? `inherit` sounds wrong at first glance—does `100%` not work?